### DISCUSSION TITLE
How to add column in Array Object?

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#2041] [Questions] [answered]
Hi I'm confused in adding Array Object in the UI table
Step1: Add the column
![image](https://user-images.githubusercontent.com/9329005/122857618-54952300-d34b-11eb-9c60-dcb03fe58258.png)

Step2: How to add other columns in UI?
- game_date type DATE
- home_team type TEXT
 visiting_team type TEXT

https://www.screencast.com/t/CgEsIbadZF

**Comment by soedirgo** [comment, marked as answer]
Hey @donPuerto, you need to Save the `game_data` column, then add other columns using either of these:
<img width="635" alt="Screenshot 2021-06-22 at 12 30 05 PM" src="https://user-images.githubusercontent.com/31685197/122863740-f02b9100-d355-11eb-90e7-f2c7c70eafa5.png">

**Reply by donPuerto** [reply]
@soedirgo After saving,  how to relate the **newly added column** to **game_data** column?

**Reply by soedirgo** [reply]
Can you elaborate what you mean by relate? Do you have a specific thing you want to do, e.g. select rows based on their `game_date`?

**Reply by donPuerto** [reply]
@soedirgo We are creating Json Object "game_data" column right?
We need to add more columns for "game_data" the question is how?
game_date type DATE
home_team type TEXT
visiting_team type TEXT

So, 
game_data : [{
   game_date: "",
   home_team: ""
   visiting_team: ""
   
]

You said just add the column...
After i add these columns, how do I relate these columns to "game_data"
game_date type DATE
home_team type TEXT
visiting_team type TEXT

**Reply by soedirgo** [reply]
Ah, so `game_data` is a separate column (see [this](https://supabase.io/docs/guides/database/json#insert-data-into-the-table) example content). It doesn't contain values for `game_date`, `home_team`, or `visiting_team`.

**Reply by donPuerto** [reply]
I get it now, this is only of holder of json data
